utils/utils.hpp

```diff
@@ -213,5 +264,35 @@ class Utils
 
     return 0;
   }
+
+  /**
+   * Fills a vector with paths to all files in directory.
+   *
+   * @param path Path to Directory.
+   * @param pathVector A vector of type filesystem::path, which will be filled
+   *                   paths for all files / folders in given directory path.
+   * @param absolutePath Boolean to determine if path is absolute or relative.
+   */
+  static void ListDir(const std::string& path,
+                      std::vector<boost::filesystem::path>& pathVector,
+                      const bool absolutePath = false)
+  {
+    if (Utils::PathExists(path, absolutePath))
+    {
+      boost::filesystem::path directoryPath(path);
+
+      // Fill the path vector with respective paths.
+      std::copy(boost::filesystem::directory_iterator(directoryPath),
+          boost::filesystem::directory_iterator(),
+          std::back_inserter(pathVector));
+
+      // Sort the path vector.
+      std::sort(pathVector.begin(), pathVector.end());
+    }
+    else
+    {
+      mlpack::Log::Warn << "The " << path << "Doesn't exist." << std::endl;
```

Comment:
```suggestion
      mlpack::Log::Warn << "The " << path << " doesn't exist." << std::endl;
```

Reply:
Right, we fix it in the next commit after I rebase. We'll have a green build. Thanks a lot.